CHANGES.md

```diff
@@ -9,6 +9,22 @@ Note to self: Breaking changes must increment either
 
 -->
 
+## 0.30.0 (2024-07-02)
+
+_**Breaking**_ ⚠️
+
+> No breaking changes were introduced in this version.
+
+_**Features**_
+
+- feat: add validator for trx addresses by @msamsami in [#384](https://github.com/python-validators/validators/pull/384)
+
+_**Maintenance**_
+
+- maint: bump version by @msamsami in [#384](https://github.com/python-validators/validators/pull/384)
+
+**Full Changelog**: [`0.29.0...0.30.0`](https://github.com/python-validators/validators/compare/0.29.0...0.30.0)
+
```

Comment:
Please add a separator after line `27` and after line `43` (like on [line 120](https://github.com/python-validators/validators/pull/384/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R120
)), to group `patch`ed changes together.

```diff
**Full ...

+---

# 0. ...
```